Source/WebKit/Shared/JavaScriptEvaluationResult.cpp

```diff
@@ -206,9 +214,9 @@ JavaScriptEvaluationResult::JavaScriptEvaluationResult(JSGlobalContextRef contex
     m_nullObjectID = std::nullopt;
 }
 
-JSValueRef JavaScriptEvaluationResult::toJS(JSGlobalContextRef context, Variant&& root)
+JSValueRef JavaScriptEvaluationResult::toJS(JSGlobalContextRef context, Value&& root)
 {
-    return WTF::switchOn(WTFMove(root), [&] (NullType) -> JSValueRef {
+    return WTF::switchOn(WTFMove(root), [&] (EmptyType) -> JSValueRef {
```

Comment:
This is causing most of the failures in glib api tests. The empty type is ignored and Null value is always returned. Creating an undefined value when EmptyType is Undefined fixes some of the failures. I'm still investigating the others.